### PR TITLE
[Metricbeat] Update Cockroachdb Overview and add SQL dashboard

### PR DESCRIPTION
**Pull request by sayden**
## What does this PR do?

In CockroachDB Metricbeat module:

- Clean / Improve visualization titles in "Overall" dashboard.
- Create new "SQL" dashboard
- Remove Cluster filter that wasn't often used.

Checks:
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- ~~[x] I have added tests that prove my fix is effective or that my feature works~~
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

@sorantis you might want to quickly take a look at the screenshots

- [x] Confirm that the filter is removed as expected.
- [ ] Confirm that overall look is the expected.

## Related issues

- Closes https://github.com/elastic/beats/issues/22981

### Overview
image

### SQL
image

### CODE REVIEW

**Review by jsoriano — COMMENTED** [review]
Looking good, only a couple of comments.

File: x-pack/metricbeat/module/cockroachdb/_meta/kibana/7/dashboard/Metricbeat-cockroachdb-overview.json [hunk 1]
```diff
@@ -893,21 +1027,575 @@
...
+                "formatter": "s,h,2",
+                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                "label": "Cockroach Process Uptime",
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "field": "prometheus.metrics.sys_uptime",
+                    "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                    "type": "max"
+                  }
```
[the remark this answers is not in the document]
Comment (sorantis) [reply x-pack/metricbeat/module/cockroachdb/_meta/kibana/7/dashboard/Metricbeat-cockroachdb-overview.json:1186]: Good question, and not sure whether average would be more appropriate. The dashboard was built by a CRDB product manager so I trust it captures the necessary information :)

**Comment by sorantis** [comment]
@sayden thanks for making a PR. The dashboards look good. One nit though, could we add navigation to link the two dashboards? It'll be more convenient for the user if the dashboards can cross link

**Comment by sayden** [comment]
Okay! screenshots updated too

File: x-pack/metricbeat/module/cockroachdb/_meta/kibana/7/dashboard/Metricbeat-cockroachdb-overview.json [hunk 2]
```diff
@@ -30,10 +32,12 @@
             "panelIndex": "1",
             "panelRefName": "panel_0",
             "title": "Number of SQL connections",
-            "version": "7.1.0"
+            "version": "8.0.0-SNAPSHOT"
```
[the remark this answers is not in the document]
Comment (sayden) [reply x-pack/metricbeat/module/cockroachdb/_meta/kibana/7/dashboard/Metricbeat-cockroachdb-overview.json]: You are right, I have re-done them in `7.11`

**Review by sorantis — APPROVED** [review]
LGTM!

**Comment by sayden** [comment]
CI errors are related. Looking into them

**Review by jsoriano — APPROVED** [review]

**Comment by sayden** [comment]
Errors are unrelated

**Pull request merged**